exp/operator/main.go

```diff
@@ -17,69 +17,176 @@ limitations under the License.
 package main
 
 import (
+	"context"
 	"flag"
+	"fmt"
+	"math/rand"
+	"net/http"
 	"os"
+	"time"
 
+	"github.com/spf13/pflag"
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/klog"
 	"k8s.io/klog/klogr"
```

Comment:
Just noticed on second viewing:
```suggestion
	"k8s.io/klog/v2/klogr"
```

Reply:
We need a linter 🙃